### PR TITLE
resolved comments, and added isomorphic fetch dep

### PR DESCRIPTION
**Pull request by sxu1998**
resolved comments on the previous merged PR, and added isomorphic fetch dependency
skipped all the integration tests for now since the clouddev CA env is down
also updated to most recent version of the js client core lib

### CODE REVIEW

File: src/ClientBase.ts [hunk 1]
```diff
@@ -60,7 +54,7 @@ export class RepositoryApiClient implements IRepositoryApiClient {
       fetch,
     };
     this.baseUrl = baseUrlDebug ?? '';
-    this.attributesClient = new AttributesClient(this.baseUrl, http);
+    this.attributesClient = new AttributesClientEx(this.baseUrl, http);
```
Comment (terrencejames-lf) [inline src/ClientBase.ts:57]: Instead of creating sub classes for all the Clients is it possible to just create extend the existing generated classes?

Comment (sxu1998) [reply src/ClientBase.ts:57]: sorry im not rlly sure what you mean, may you give me an example?

Comment (sxu1998) [reply src/ClientBase.ts:57]: i guess we can save this for another time -> https://tfs/DefaultCollection/Server%20Tools/_workitems/edit/382388

**Review by lf-jason-jia — DISMISSED** [review]

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ on:
   workflow_dispatch:
 
 env:
-  NPM_VERSION: "1.0.4"
+  NPM_VERSION: "1.0.3"
```
Comment (terrencejames-lf) [inline .github/workflows/main.yml:13]: I believe 1.0.3 was already published

File: src/ClientBase.ts [hunk 3]
```diff
@@ -150,3 +143,930 @@ function isRetryable(response: Response, init: RequestInit): boolean {
...
+function CreateMaxPageSizePreferHeaderPayload(maxSize?: number): string | undefined {
+  //puts the max size into the prefer header of the GET request
+  if (!maxSize) {
+    return undefined;
+  } else {
+    return `maxpagesize=${maxSize}`;
+  }
+}
+
+export interface IAttributeClient extends generated.IAttributesClient {
```
Comment (terrencejames-lf) [inline src/ClientBase.ts:179]: So you don't need to extend the interface here, just keep it the same (IAttributesClient)

Comment (sxu1998) [reply src/ClientBase.ts]: ok done

File: .github/workflows/main.yml (hunk 2 above)
Comment (sxu1998) [reply .github/workflows/main.yml]: 1.0.3 -> 1.0.4

**Review by terrencejames-lf — APPROVED** [review]

**Pull request merged**